tools/watch_logs.py

```diff
@@ -45,7 +45,7 @@ def logs_from_config(supervisor_conf):
 watched.append('log/nginx-bad-access.log')
 watched.append('log/nginx-error.log')
 watched.append('log/fake_oauth2.log')
-
+watched.append('log/app_00.log')
```

Comment:
Do you want to watch other files too?

Reply:
I think there's a bigger problem we need to address here, which is that we should pick up on new logfiles as they appear and add threads for them.  I'll fix that in another PR.